profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestEntitlement.java

```diff
@@ -860,7 +860,7 @@ public void testCreateSubscriptionBillingInThePast() throws Exception {
 
 
         Assert.assertEquals(entitlementJson.getState(), EntitlementState.ACTIVE);
-        verifyChargedThroughDate(entitlementJson.getSubscriptionId(), new LocalDate("2012-05-24"));
```

Comment:
This unlikely needs to change -- if the assertion failed, it was probably a flaky test run.